scripts/action-infra-stack.sh

```diff
@@ -53,8 +51,8 @@ else
   fi
 fi
 
-if [ -z "$TERRAFORM_WORKSPACE_NAME" ] ; then
-  echo Set TERRAFORM_WORKSPACE_NAME
+if [ -z "$WORKSPACE" ] ; then
```

Comment:
Is it worth updating the comments at top of script to be clear about the inputs etc

Reply:
Done